bespin/option_spec/bespin_specs.py

```diff
@@ -302,8 +284,8 @@ def stack_spec(self):
             , stack_json = valid_stack_json(default="{config_root}/{_key_name_1}.json")
             , stack_yaml = valid_stack_yaml(default="{config_root}/{_key_name_1}.yaml")
 
-            , params_json = valid_params_json(default="{config_root}/{environment}/{_key_name_1}-params.json")
-            , params_yaml = valid_params_yaml(default="{config_root}/{environment}/{_key_name_1}-params.yaml")
```

Comment:
I'm still confused.

If only the json file exists and neither are specified then params_yaml will be NotSpecified
if only the yaml file exists and neither are specified then params_json will be NotSpecified
if both the yaml and the json file exist then you have a confusing setup

if you define them in your configuration, you probably shouldn't have it also as a file somewhere.